pdfshuffler/pdfshuffler.py

```diff
@@ -519,18 +519,20 @@ def export_to_file(self, file_out, only_selected=False):
         pdf_output = PdfFileWriter()
         pdf_input = []
         for pdfdoc in self.pdfqueue:
-            pdfdoc_inp = PdfFileReader(file(pdfdoc.copyname, 'rb'))
+            pdfdoc_inp = PdfFileReader(open(pdfdoc.copyname, 'rb'))
```

Comment:
Could you use a context manager here?

Reply:
While using a context manager looks like a good idea in that case it's tricky because the object pdfdoc_inp is used ouside of the for loop so in theory the context manager would have to cover most of the function, but the name is only know inside the loop. I think it's better to leave it that way.

Reply:
A nasty case. Thanks for looking into it.